ext/st/module.lb

```diff
@@ -128,7 +128,7 @@ def build(env):
     env.substitutions = bprops
     env.substitutions.update({
         "headers": [bprops["device_header"], bprops["system_header"]],
-        "defines": [bprops["define"]],
+        "defines": [bprops["define"], "__PROGRAM_START"],
```

Comment:
This hotfixes this issue in CMSIS5: https://github.com/ARM-software/CMSIS_5/issues/617